parsers/meriyah.ts

```diff
@@ -0,0 +1,36 @@
+// This module is suitable for passing as options.parser when calling
+// recast.parse to process JavaScript code with Meriyah:
+//
+//   const ast = recast.parse(source, {
+//     parser: require("recast/parsers/meriyah")
+//   });
+//
+import { getOption } from "../lib/util";
+
+function convertComment(comment: any): any {
+  let { type } = comment;
+  type = type[0] === "S" ? "Line" : "Block"; // SingleLine/MultiLine
```

Comment:
Are the anticipated values of `type` literally 'SingleLine' and 'MultiLine'?

If so, clearest to directly say `type === 'SingleLine' ? …`, rather than look at `type[0]`.

Reply:
Yes, will do.